velox/substrait/tests/SubstraitVeloxPlanConverterTest.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/Random.h>
+#include <random>
+
+#include "velox/dwio/dwrf/test/utils/BatchMaker.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/vector/tests/VectorMaker.h"
+
+#include "velox/substrait/SubstraitToVeloxPlan.h"
+#include "velox/substrait/VeloxToSubstraitPlan.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+using namespace facebook::velox::substrait;
+
+// using facebook::velox::test::BatchMaker;
+
+class SubstraitVeloxPlanConvertorTest : public OperatorTestBase {
+ protected:
+  /// the function used to make data
+  /// \param size the number of RowVectorPtr.
+  /// \param childSize the size of rowType_, the number of columns
+  /// \param batchSize batch size.  row number
+  /// \return std::vector<RowVectorPtr>
+  std::vector<RowVectorPtr>
+  makeVector(int64_t size, int64_t childSize, int64_t batchSize) {
+    std::vector<RowVectorPtr> vectors;
+    for (int i = 0; i < size; i++) {
+      std::vector<VectorPtr> children;
+      std::mt19937 gen(std::mt19937::default_seed);
+      for (int j = 0; j < childSize; j++) {
+        std::vector<int32_t> childVector(
+            batchSize, static_cast<int32_t>(folly::Random::rand32(gen)));
+        VectorPtr child =
+            createSpecificScalar<int32_t>(batchSize, childVector, *pool_);
```

Comment:
You can use an existing helper function makeFlatVector:

```
makeFlatVector<int32_t>(batchSize, [&] (auto row) { return folly::Random::rand32(gen); })
```

Reply:
done.